_data/sections/contribution-limits/legends.yml

```diff
@@ -88,3 +88,8 @@
       max: Infinity
       label: Unlimited
       color: "#cb181d"
+- name: Party2Party
+  type: singular
+  scale:
+    - label: "Forbidden but funds get transferred between the two entities following internal procedures. And this is a sentence to act as a space filler."
```

Comment:
Is this intended to be included in the merge? "And this is a sentence to act as a space filler"